scss/styles.scss

```diff
@@ -0,0 +1,296 @@
+@import "_reset";
+@import "_mixins";
+@import "_fonts";
+
+//variables
+$main-gray: #626262;
+$support_gray: #f3f3f3;
+$aside: #b4ad9e;
+
+
+$firstBreakPoint: 600px;
+$secondBreakPoint: 900px;
+$thirdBreakPoint: 1200px;
+
+
+
+
+body{
+  font-family: Playfair, Arial, sans-serif;
+  font-size: 16px;
+}
+.main-content-wrapper{
+  padding-top: 56px;
+}
+
+.wrapper{
+  max-width: 1200px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+/**********
+HEADER styles
+***********/
+
+.dropdown{
+  position: absolute;
+  top: -500px;
+  right: -500px;
+  transition: .4s;
+  @include mQ($secondBreakPoint) {
+    position: static;
+
+  }
+}
+
+.header-wrapper{
+  background: white;
+  position: fixed;
+  display:flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: .5em;
+  @include mQ($thirdBreakPoint){
+    position:static;
+  }
+  .main-logo{
+    display: block;
+    max-width:119px;
+    max-height:23px;
+    width: auto;
+    height: auto;
+  }
+  .main-nav{
+    display:flex;
+  }
+  .main-menu {
+    display:block;
+    background-color: white;
+    flex-direction: column;
+    .main-menu-item{
+      a{
+        display:inline-block;
+        padding: 3vh 16vw;
+        text-transform: uppercase;
+        @include mQ($secondBreakPoint) {
+          padding: .5em 2em;
+        }
+      }
+      &:hover{
+        background-color: $main-gray;
+        color: white;
+        transition: .5s;
+      }
+    }
+    @include mQ($secondBreakPoint) {
+      display: flex;
+      flex-direction: row;
+    }
+  }
+  .hamburger-wrapper{
+    padding-top: 4px;
+    .hamburger{
+      @include mQ($secondBreakPoint) {
+        display: none;
+      }
+    }
+    &:hover{
+      .dropdown{
+      top: 0;
+      right: 0;
+      transition: .8s;
+      }
+    }
+  }
+}
+
+
+/**********
+ARTICLE styles
+***********/
+
+.main-article,
+.other-article{
```

Comment:
Artykuły mają większe odstępy pomiędzy artykułami i contentem sąsiadującymi z góry / dołu. Szczegół ale to pierwsza rzecz, na którą zwraca uwagę Klient